### PR TITLE
Updated protobuf definitions

### PR DESCRIPTION
**Pull request by rene-m-hernandez**
Updates to latest Envoy API (as of Sept. 13)

Followed these instructions: https://github.com/envoyproxy/java-control-plane#bring-api-up-to-date-with-data-plane-api

### CODE REVIEW

**Comment by rene-m-hernandez** [comment]
When attempting to make the update to the latest Envoy API definitions, a couple of issues were encountered:

1. A dependency on a previously unused proto was added
    Solved: Modify tools/update-api.sh to copy the missing file 
1. v2 of the Envoy API was copied to v3alpha as part of the UDPA work being done. This is causing a conflict at compilation as multiple proto files will attempt to generate the same Java class. 
    No solution at the moment.

**Comment by rene-m-hernandez** [comment]
@jakubdyszkiewicz @snowp @joeyb Any chance I can get a review and get a new version cut with this merged?

File: api/pom.xml [hunk 1]
```diff
@@ -58,7 +58,9 @@
                     <protocArtifact>com.google.protobuf:protoc:${protobuf.version}:exe:${os.detected.classifier}</protocArtifact>
                     <pluginId>grpc-java</pluginId>
                     <pluginArtifact>io.grpc:protoc-gen-grpc-java:${grpc.version}:exe:${os.detected.classifier}</pluginArtifact>
-
+                    <excludes>
+                        <exclude>**/v3alpha/*</exclude>
```
Comment (snowp) [inline api/pom.xml]: do we still need this?

Comment (rene-m-hernandez) [reply api/pom.xml]: Yes, this is needed as long as this folder exists: https://github.com/envoyproxy/envoy/tree/706f910954574e5d18dd4ba94ce66e8ee77b238c/api/envoy/api/v3alpha

Comment (rene-m-hernandez) [reply api/pom.xml]: No, not needed. Removed.

**Comment by rene-m-hernandez** [comment]
Thank you for merging @snowp! Any chance we can release a new version with the updated APIs?

**Comment by snowp** [comment]
Sorry about the delay, let me try to kick off a release

**Pull request merged**